op-enable.go

```diff
@@ -250,6 +250,10 @@ func composeUp(d driver.DistroDriver, json map[string]interface{}, env map[strin
 	defer os.Unsetenv("COMPOSE_HTTP_TIMEOUT")
 	defer os.Unsetenv("DOCKER_CLIENT_TIMEOUT")
 
+	// set docker-compose.yml path to share directory with docker-compose container.
+	os.Setenv("COMPOSE_FILE", ymlPath)
```

Comment:
Thanks for the PR, I'm not aware how is this different than passing compose file with `-f ymlPath`.